nwb_conversion_tools/json_schema_utils.py

```diff
@@ -28,7 +28,7 @@ def get_base_schema(tag=None, root=False, id_=None, **kwargs) -> dict:
         required=[],
         properties={},
         type='object',
-        additionalProperties=False
+        additionalProperties=True
```

Comment:
Can you explain why this flag needs to be flipped?

Reply:
In my case, I believe I would not be able to specify `Electrodes` without that flag being True, because it is not in the schema. Of course it could be added to it instead. 

```
"Ecephys": {
    "required": [
      "ElectrodeGroup",
      "Device"
    ],
    "properties": {
      "Device": {
        "type": "array",
        "minItems": 1,
        "items": {
          "$ref": "#/properties/Ecephys/properties/definitions/Device"
        },
        "default": [
          {
            "name": "Axona",
            "description": "Axona DacqUSB, sw_version=1.2.2.16",
            "manufacturer": "Axona"
          }
        ]
      },
      "ElectrodeGroup": {
        "type": "array",
        "minItems": 1,
        "items": {
          "$ref": "#/properties/Ecephys/properties/definitions/ElectrodeGroup"
        },
        "default": [
          {
            "name": "Group0",
            "location": "",
            "device": "Axona",
            "description": "Group 0 electrodes."
          }
        ]
      },
      "ElectricalSeries": {
        "required": [
          "name"
        ],
        "properties": {
          "name": {
            "description": "The name of this TimeSeries dataset",
            "type": "string",
            "default": "ElectricalSeries"
          },
          "resolution": {
            "description": "The smallest meaningful difference (in specified unit) between values in data",
            "type": "number",
            "default": -1.0
          },
          "conversion": {
            "description": "Scalar to multiply each element in data to convert it to the specified unit",
            "type": "number",
            "default": 1.0
          },
          "starting_time": {
            "description": "The timestamp of the first sample",
            "type": "number"
          },
          "rate": {
            "description": "Sampling rate in Hz",
            "type": "number"
          },
          "comments": {
            "description": "Human-readable comments about this TimeSeries dataset",
            "type": "string",
            "default": "no comments"
          },
          "description": {
            "description": "Description of this TimeSeries dataset",
            "type": "string",
            "default": "Raw acquisition traces."
          }
        },
        "type": "object",
        "additionalProperties": false,
        "tag": "pynwb.ecephys.ElectricalSeries"
      },
      "definitions": {
        "Device": {
          "required": [
            "name"
          ],
          "properties": {
            "name": {
              "description": "the name of this device",
              "type": "string"
            },
            "description": {
              "description": "Description of the device (e.g., model, firmware version, processing software version, etc.)",
              "type": "string"
            },
            "manufacturer": {
              "description": "the name of the manufacturer of this device",
              "type": "string"
            }
          },
          "type": "object",
          "additionalProperties": false,
          "tag": "pynwb.device.Device"
        },
        "ElectrodeGroup": {
          "required": [
            "device",
            "description",
            "name",
            "location"
          ],
          "properties": {
            "name": {
              "description": "the name of this electrode",
              "type": "string"
            },
            "description": {
              "description": "description of this electrode group",
              "type": "string"
            },
            "location": {
              "description": "description of location of this electrode group",
              "type": "string"
            },
            "device": {
              "description": "the device that was used to record from this electrode group",
              "type": "string",
              "target": "pynwb.device.Device"
            }
          },
          "type": "object",
          "additionalProperties": false,
          "tag": "pynwb.ecephys.ElectrodeGroup"
        }
      }
    },
    "type": "object",
    "additionalProperties": true,
    "tag": "Ecephys"
  }
```

Reply:
If it's something required for a specific interface, it should be flipped within that interface. Given this is such a core function, there has to be a good general reason for flipping it (which there could very well be, but one example isn't enough for that). Note this isn't just for metadata schemas either, but anything that ever calls the base schema retrieval.

Reply:
@sbuergers @CodyCBakerPhD I accepted Cody's turning that back to False, I can't remember anymore why I needed that as True. We can change it back in the next PR if we spot a reason why